### DISCUSSION TITLE
Openmetadata can't ingest sample data and metadata from the Oracle database

### DISCUSSION THREAD

**Discussion by vulong1503** [open-metadata/OpenMetadata#16793] [Q&A]
Hi everyone,
I'm quite new to Openmetadata. i'm trying to connect my Oracle Database 19c to Openmetadata. I have followed the step in the https://docs.open-metadata.org/v1.4.x/deployment/ingestion document, granted all needed privileges. 
![image](https://github.com/open-metadata/OpenMetadata/assets/75939336/e7ff4042-cbad-4fca-98a9-df2319f21c06)

However, I couldn't ingest any sample data from the HR schema (the sample schema from Oracle) after run the DAG for a long time. 
Here is the schema in my database
![image](https://github.com/open-metadata/OpenMetadata/assets/75939336/48e1df22-703e-4777-b109-aafdb3d83847)
 
I only got the metadata for COUNTRIES table:
![image](https://github.com/open-metadata/OpenMetadata/assets/75939336/6a897b1a-154a-41c9-9b92-1ff5af98dc88)

Also, after the metadata ingestion pipeline completed, I ran the profiler ingestion and it ran so long without update any sample data to Openmetadata.

How can I fix these problems and get the data into Openmetadata?
Thanks and best regards to everyone

**Comment by Prajwal214** [comment]
@vulong1503 Ensure you have the necessary permissions, including select_catalog_role and create session. Also, verify that the profiler pipeline has the generateSampleData flag enabled.

![image](https://github.com/open-metadata/OpenMetadata/assets/167504578/7504d2b6-cca5-4e22-99c8-94d9d371f671)

**Reply by xiaoxin05** [reply]
please slove this problem？about this profilter after work ，i don't have any data in Table profile/insert_update data chanage？

**Reply by Prajwal214** [reply]
Please reach out in [slack support](https://slack.open-metadata.org/)
